### PR TITLE
Release changes for 3.2.0

### PR DESCRIPTION
**Pull request by Jennifer-John**
# Description
Release changes for 3.2.0
Document Updates and changelog

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| N/A |

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [X] I have performed Ansible Sanity test using --docker default
- [] I have verified that new and existing unit tests pass locally with my changes
- [] I have not allowed coverage numbers to degenerate
- [] I have maintained at least 90% code coverage
- [] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [] I have added tests that prove my fix is effective or that my feature works
- [X] Backward compatibility is not broken

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: changelogs/.plugin-cache.yaml [hunk 1]
```diff
@@ -31,6 +31,21 @@ plugins:
       name: email
       namespace: ''
       version_added: 1.5.0
+    file_dns:
+      description: Manage File DNS for PowerStore
+      name: file_dns
+      namespace: ''
+      version_added: 3.1.0
```
Comment (Bhavneet-Sharma) [inline changelogs/.plugin-cache.yaml:38]: I guess these changed need to update to their respective correct versions.

Comment (sachin-apa) [reply changelogs/.plugin-cache.yaml:38]: this is a plugin cache file do you see it incorrect in changelog.yml as well ?, if can point out the correct version i can update,.
FYI, they are unchanged from previous release.

Comment (sachin-apa) [reply changelogs/.plugin-cache.yaml:38]: this is a plugin cache file, please do you see it in changelog.yml/changelor.rst as well ?, if you can point out the correct version i can update and comment in changelog.yml i can update,.
FYI, they are unchanged from previous release.

Comment (Bhavneet-Sharma) [reply changelogs/.plugin-cache.yaml:38]: Sure,
I'll share the correct data.

**Review by Bhavneet-Sharma — DISMISSED** [review]

**Review by Bhavneet-Sharma — DISMISSED** [review]

File: docs/modules/certificate.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ The below requirements are needed on the host that executes this module.
 
 - A Dell PowerStore storage system version 3.0.0.0 or later.
 - Ansible-core 2.13 or later.
```
Comment (trisha-dell) [inline docs/modules/certificate.rst:24]: 2.14. Update all RST accordingly

**Review by sachin-apa — APPROVED** [review]

**Review by felixs88 — APPROVED** [review]

File: plugins/doc_fragments/powerstore.py [hunk 3]
```diff
@@ -51,7 +51,7 @@ class ModuleDocFragment(object):
     requirements:
       - A Dell PowerStore storage system version 3.0.0.0 or later.
       - Ansible-core 2.13 or later.
```
Comment (trisha-dell) [inline plugins/doc_fragments/powerstore.py:53]: 2.13 - > 2.14

**Review by meenakshidembi691 — APPROVED** [review]

File: docs/modules/certificate.rst (hunk 2 above)
Comment (sachin-apa) [reply docs/modules/certificate.rst]: Done

File: plugins/doc_fragments/powerstore.py (hunk 3 above)
Comment (sachin-apa) [reply plugins/doc_fragments/powerstore.py]: Done

**Review by Bhavneet-Sharma — APPROVED** [review]

**Review by trisha-dell — APPROVED** [review]

**Review by sachin-apa — APPROVED** [review]

**Review by felixs88 — APPROVED** [review]

**Review by meenakshidembi691 — APPROVED** [review]

**Pull request merged**